### DISCUSSION TITLE
trying to get token with tesla_auth: Access denied

### DISCUSSION THREAD

**Discussion by brianmay** [teslamate-org/teslamate#3554] [General]
Using the client https://github.com/adriankumpf/tesla_auth version 0.9.0:

```
Access Denied

You don't have permission to access "http://auth.tesla.com/oauth2/v3/authorize?" on this server.
Reference #18.3f5ad617.1702073152.188d242d
```

Is this just me? Or Tesla?

**Comment by NirKli** [comment]
> Using the client https://github.com/adriankumpf/tesla_auth version 0.9.0:
> 
> ```
> Access Denied
> 
> You don't have permission to access "http://auth.tesla.com/oauth2/v3/authorize?" on this server.
> Reference #18.3f5ad617.1702073152.188d242d
> ```
> 
> Is this just me? Or Tesla?

I just tried it and it worked for me on Windows 11.
Have you attempted any other methods to retrieve the Tesla token to check if Tesla has blocked you?

**Reply by brianmay** [reply]
Curious thing though it happens before I even enter my email address.

I guess they might have blocked my IP address though. But really? 

For a while now, `tesla_mate` is the only method that works.

Hmmm. Android app works. How odd. But painful, I need to get the token on to my Linux box now.

**Comment by brianmay** [comment]
I filled a bug report: https://github.com/adriankumpf/tesla_auth/issues/67
